### ISSUE TITLE
AllMusic.com

### ISSUE THREAD

**Issue by donwiss** [uBlockOrigin/uAssets#171]
### URL(s) where the issue occurs

http://www.allmusic.com/

### Describe the issue

The following message is displayed:

Help Support AllMusic
It appears that you are using an AdBlocker or something is blocking advertisements from being displayed on our site. This impacts our ability to keep AllMusic up and running. Please help us keep our content free by selecting one of the following options:
CONTINUE WITH SUPPORTFREECONTENT
Access AllMusic ad-free with the SupportFreeContent extension.
(More info)
CONTINUE AFTER DISABLING AD BLOCKER
Allow ads on AllMusic by whitelisting this site or disabling your ad blocker.
(Help)
CONTINUE WITH A PAID SUBSCRIPTION
Support AllMusic for a faster, ad-free experience with a $12 annual subscription.

### Screenshot(s)

The above is the full text/

### Versions

- Browser/version: Firefox 49.0.2
- uBlock Origin version: 1.9.12

### Settings

- Initially all default settings. I added some custom filters and unchecked some boxes. Still no good. I'd prefer to return to all default settings and use the included filters.

### Notes

I posted here:
https://github.com/reek/anti-adblock-killer/issues/1941#issuecomment-255449358

**Comment by svnpenn** [comment]
https://github.com/gorhill/uBlock/issues/2095

**Comment by svnpenn** [comment]
https://github.com/reek/anti-adblock-killer/issues/2020

**Comment by gorhill** [comment]
Fixed with 3e000caba53d29cfe06c70c23bda116d238a53d5.

**Comment by svnpenn** [comment]
Confirmed fixed, thank you

**Comment by svnpenn** [comment]
@gorhill this is broken again

**Comment by donwiss** [comment]
AllMusic is at it again. Not on the home page, but if you go to an artist or album page they have this Saambaa ad box that displays ads for local performances. (At least it is there if you live in a major city.) Initially if you left it playing on your screen your screen saver wouldn't work.

In Firefox I went to about:config. I searched on media.autoplay.enabled. And toggled the True value to False. I did post in AllMusic's forum that I was annoyed at the screen saver issue and how I fixed it.

That worked for about two weeks. Then it started up again. At least now it seems the screen saver will work.

It would be better if one could completely block this.
